server/channels/app/platform/web_hub.go

```diff
@@ -265,11 +273,17 @@ func (ps *PlatformService) WebConnCountForUser(userID string) int {
 }
 
 // Register registers a connection to the hub.
-func (h *Hub) Register(webConn *WebConn) {
+func (h *Hub) Register(webConn *WebConn) error {
+	wr := &webConnRegisterMessage{
+		conn: webConn,
+		err:  make(chan error),
```

Comment:
It looks safe as is, but how about making this buffered to avoid blocking the hub even for a moment?

Reply:
This feels like a micro-optimization. I want to lean on the side of correctness for now.